lib/plaid/connect.rb

```diff
@@ -44,7 +44,7 @@ def self.run(http, request)
       response = http.request(request)
 
       if response.body.nil? || response.body.empty?
-        raise Plaid::PlaidError.new(0, 'Server error', 'Try to connect later')
+        raise(PlaidServerError, 'Server error, try to connect again later.')
```

Comment:
Parens are not needed here. Check the [style guide](https://github.com/bbatsov/ruby-style-guide#exceptions).